lib/CMakeLists.txt

```diff
@@ -37,13 +37,23 @@ else()
     set(LIBCPP_PCP_CLIENT_INSTALL_DESTINATION lib)
 endif()
 
-
 add_library(libcpp-pcp-client-src OBJECT ${SOURCES})
 set_target_properties(libcpp-pcp-client-src PROPERTIES POSITION_INDEPENDENT_CODE true)
 add_library(libcpp-pcp-client SHARED $<TARGET_OBJECTS:libcpp-pcp-client-src>)
 target_link_libraries(libcpp-pcp-client PRIVATE ${LIBS} ${PLATFORM_LIBRARIES})
 set_target_properties(libcpp-pcp-client  PROPERTIES PREFIX "" SUFFIX ".so" IMPORT_PREFIX "" IMPORT_SUFFIX ".so.a")
 
+# Generate the export header
+include(GenerateExportHeader)
+generate_export_header(libcpp-pcp-client EXPORT_FILE_NAME "${CMAKE_CURRENT_LIST_DIR}/inc/cpp-pcp-client/export.h")
+
+if ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux" OR "${CMAKE_SYSTEM_NAME}" MATCHES "AIX")
```

Comment:
This appears to be safe on AIX, but doesn't actually do anything. Investigating why CMake on AIX doesn't try to set visibility.